### DISCUSSION TITLE
RedHat 9, CentOS9 RockyLinux 9

### DISCUSSION THREAD

**Discussion by tomu84** [rspamd/rspamd#4331] [General]
When RSPAMD will be compatible with CenOS 9 ? 
Now i have a problem with "libicu". Rspamd requires version 60 which is deprecated.

**Comment by D3an1el** [comment]
Any news whether this will happen? you have 7 &  8 already.

**Comment by vstakhov** [comment]
Rspamd is available for RHEL 9 and it's derivatives.
